src/components/Media/Media.tsx

```diff
@@ -14,7 +14,6 @@ interface Props {
 export function Media({ className, image, style, title }: Props) {
     const computedClassName = classNames('prezly-slate-media', className, {
         'prezly-slate-media--image': !image.isGif(),
-        'prezly-slate-media--video': image.isGif(),
```

Comment:
By the way, removing classes here is potentially dangerous. Because they are global, there's a chance a theme implementation uses these global classes to style content.